.github/workflows/build.yml

```
@@ -86,7 +86,9 @@ jobs:
       run: cmake --build ${{github.workspace}}/build -- all examples
 
     - name: Test
-      # working-directory: ${{github.workspace}}/build
       # Execute tests defined by the CMake configuration.
       # See https://cmake.org/cmake/help/latest/manual/ctest.1.html for more detail
-      run: cd ${{github.workspace}}/build && ninja check
+      run: |
+        cd ${{github.workspace}}/build
+        ninja tests_build
+        ninja test
```

```suggestion
        ninja tests
        ninja run_tests
```
What about that? we `make tests` to build them, and we `make run_tests` to run them.

---

good point, but `test` is a default target generated by cmake so we cannot change that